frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -218,12 +228,23 @@
     }
 
     function exportExcel(){
+        $('#keywordsearchinput').val("");
+        useSearch('keywordsearchinput');
+
         var portletURL = PortletURL.createURL('<%= PortletURLFactoryUtil.create(request, portletDisplay.getId(), themeDisplay.getPlid(), PortletRequest.RESOURCE_PHASE) %>')
                 .setParameter('<%=PortalConstants.ACTION%>', '<%=PortalConstants.EXPORT_TO_EXCEL%>');
-        portletURL.setParameter('<%=PortalConstants.KEY_SEARCH_TEXT%>', $('#keywordsearchinput').val());
+        portletURL.setParameter('<%=PortalConstants.KEY_SEARCH_FILTER_TEXT%>', $('#keywordsearchinput').val());
```

Comment:
Why do you pass this value if it is set to `""`? Should `PortalConstants.KEY_SEARCH_FILTER_TEXT` be removed completly, looks unused as you remove the corresponding **keep-after-reload** functionality?